### PR TITLE
SG-22646-shotgrid Shotgun instances has been replaced

### PR DESCRIPTION
**Pull request by zamu5**

### CODE REVIEW

File: python/shotgun_model/shotgun_query_model.py [hunk 1]
```diff
@@ -437,7 +437,7 @@ def _update_item(self, item, data_item):
         """
         raise NotImplementedError(
             "The '_update_item' method has not been "
-            "implemented for this ShotgunQueryModel subclass."
+            "implemented for this ShotGridQueryModel subclass."
```
Comment (jfboismenu) [inline python/shotgun_model/shotgun_query_model.py:440]:
```suggestion
            "implemented for this ShotgunQueryModel subclass."
```

File: python/settings/user_settings.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def __init__(self, bundle):
 
         self.__fw = sgtk.platform.current_bundle()
 
-        self.__settings = QtCore.QSettings("Shotgun Software", bundle.name)
+        self.__settings = QtCore.QSettings("ShotGrid Software", bundle.name)
```
Comment (jfboismenu) [inline python/settings/user_settings.py:46]:
```suggestion
        self.__settings = QtCore.QSettings("Shotgun Software", bundle.name)
```

If we did this, then users would lose their settings.

**Review by jfboismenu — CHANGES_REQUESTED** [review]

**Review by jfboismenu — APPROVED** [review]
Looks good to me, over to you @mathurf

**Review by mathurf — APPROVED** [review]
LGTM
Tested with Shotgunpanel and demo app

**Pull request merged**